_reports/ishanSrt.md

```diff
@@ -0,0 +1,162 @@
+phase:
+  - Bonding: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/46
+  - Phase 1: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/56
+  - Phase 2: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/70
+  - Phase 3: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/72
+bio: >
+  I am a third year student (expected graduation date: May 2020) of Computer
+  Science and Engineering at Indian Institute of Technology Dharwad. I
+  participated in GSoC and worked with [coala](http://coala.io) to implement
+  the coala-quickstart Green Mode for the repository
+  [coala-quickstart](https://github.com/coala/coala-quickstart) to generate
+  project dependent configuration files for the coala project, which make
+  coala show no inconsistencies in the code base and are helpful in detecting
+  inconsistencies in future commits. I also solved and contributed small
+  patches to other coala repositories during the time period of GSoC.
+social:
+  - GitHub:
+      - username: ishanSrt
+      - link: https://github.com/ishanSrt
+  - GitLab:
+      - username: ishanSrt
+      - link: https://gitlab.com/ishanSrt
+  - Gitter:
+      - username: ishanSrt
+      - link: https://gitter.im/ishanSrt
+  - OpenHub:
+      - username: ishanSrt
+      - link: https://www.openhub.net/accounts/ishanSrt
+email: ishan.srt@gmail.com
+blog: https://www.dowhatucant.com/gsoc_archive.html
+activity:
+
+  - 1:
+      - repo: projects
+      - link: https://github.com/coala/projects/commit/e5724fa9b0be71304277e508c2f317318aeed1c4
+      - details: >
+         [Bonding Period Work] quickstart_green_mode.md: Add Quickstart Green Mode Project
+
+  - 2:
+      - repo: projects
+      - link: https://github.com/coala/projects/commit/ea3f0ac7e8d98cdf1afa5d6e99345a760f83dc39
+      - details: >
+         [Bonding Period Work] .gitignore: Add .DS_Store
+
+  - 3:
+      - repo: cEPs
+      - link: https://github.com/coala/cEPs/commit/0daa94289919d5dff171ce771b24f057c64f33d5
+      - details: >
+         README.md: Add green mode cEP
+
+  - 4:
+      - repo: cEPs
+      - link: https://github.com/coala/cEPs/commit/e58b935be77459b9323a80ef9b47267d445d8c5c
+      - details: >
+         cEP-0022.md: Add cEP for quickstart green mode
+
+  - 5:
+      - repo: cEPs
+      - link: https://github.com/coala/cEPs/commit/f423227b4868376496f2280ee5e69aeb2db66724
+      - details: >
+         CODEOWNERS: Add a list of developers
+
+  - 6:
+      - repo: coala-quickstart
+      - link: https://github.com/coala/coala-quickstart/commit/a725f019c898a3422c856e1deae9cca906594883
+      - details: >
+         green_mode.py: Run BEAR_DEPS bears while testing
+
+  - 7:
+      - repo: coala-quickstart
+      - link: https://github.com/coala/coala-quickstart/commit/9e91285ac87cd36ee846743f1c9d891c324f8c18
+      - details: >
+         bear_settings.yaml: Fix wrong bear for settings
+
+  - 8:
+      - repo: coala-quickstart
+      - link: Project.py: Fix bug while printing languages
```

Comment:
this is the cause of the Netlify breakage.  It doesnt like the ":" , but a link here will solve that.

Reply:
link? what link?